src/main/java/seedu/address/ui/ModuleListPanel2.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.events.ui.ModulePanelSelectionChangedEvent;
+import seedu.address.model.module.Module;
+
+
+/**
+ * Describes the design of second UI panel containing module list display.
+ */
+public class ModuleListPanel2 extends UiPart<Region> {
```

Comment:
Also, can we do some inheritance or such of ModuleListPanel and ModuleListPanel2?

Reply:
Nvm, lets fix this in v1.4